### DISCUSSION TITLE
Signed up to use Build Beta, it worked for one day.

### DISCUSSION THREAD

**Discussion by BreadAndButter75** [blinksh/blink#2081] [Support]
iPad app unable to use Build Mosh shortly after signing up for the build Beta. It seemed to have worked a time or two but no longer works. I consistently see this response “NoWGConfig” when I run a command like Build PS or Build Images List. I’ve confirmed that I do have the information in the config section of the app. You can see the same in the screenshot.

**Comment by carloscabanero** [comment]
Hi! This was a temporary issue but it should be fixed right now. For some reason, the DO API was not returning IPs and then the machine was not able to shut down safely (as we risked losing your data). It is now solved and you should be able to log in again and have the system as it was.

I also added you another 50 more credits for the inconvenience.

Please let me know once you read this message so we can delete it, as the screenshot has your email account. Next time please know you can contact us on support@blink.build, and we can help you faster there as well.

Thanks!

**Comment by BreadAndButter75** [comment]
Thank you. You can delete the screen shot.

Do you know of a config that allows me to run docker containers? I really
wanted to use it as a way to test containers. It seems like docker has a
requirement on IPTables that doesn’t work on MOSH. I’m also not really sure
how to use something the build commands to build containers other than
MOSH.

 I’m very new to development like this. Maybe I’m over complicating this a
lot.

**Comment by carloscabanero** [comment]
So quick note on Build, containers and usage. You can already start any container you want by just doing `build up <container_name>`, where the <container_name> would be a container from the Docker registry (ie, ubuntu). Then you can connect to it by doing `build mosh <container_name>`. You can send multiple `build up` commands if you want to.

The easiest way to start developing using Blink Build is to just do `build mosh`. When you just do `build mosh`, Blink connects you transparently to our "development container", which we call "htools". This container already has a lot of compilers, editors, runtimes, etc... built-in. You can easily do Python, Ruby, JS, Rust, Go, etc... all in there without installing or complicating yourself for anything. You can clone your repositories, and Blink will take care of saving the disk, etc... once it disconnects you. 

And it should really be just as easy as that.
